gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,187 @@
++
+The readiness probe is configured by default with the following values:
++
+[source,helm]
+----
+readinessProbe:
+ exec:
+   command:
+   - flexctl
+   - probe
+   - --check=readiness
+   - --allow-api-errors
+   - --allow-envoy-errors
+   - --allow-policy-errors
+ initialDelaySeconds: 10
+ periodSeconds: 10
+ failureThreshold: 2
+ timeoutSeconds: 5
+----
++
+[cols="1,2"]
+|===
+| Parameter | Description
+| `exec.command` | The readiness probe command
+| `--allow-api-errors`| The flag to allow errors in API instances
+| `--allow-envoy-errors`| The flag to allow Envoy configuration errors
+| `--allow-policy-errors`| The flag to allow policy configuration errors
+| `initialDelaySeconds` | The time in seconds to wait after startup before executing the first readiness probe
+| `periodSeconds` | The period in seconds between each readiness probe
+| `failureThreshold` | The number of failed readiness probes before the Kubernetes pod is restarted
+| `timeoutSeconds` | The time in seconds before the readiness probe times out
+|===
++
+To modify the default parameters, you must update your Helm chart. You can modify these parameters during or after the initial installation of the Helm chart. For more information about how to update a Helm chart, see xref:flex-gateway-k8-change-helm-settings.adoc[].
++
+NOTE: In the Helm chart used to deploy Flex Gateway, omit the `--wait` flag when using a readiness probe. Not doing so results in a readiness failure because Helm waits for a readiness state of `true`.
++
+[discrete]
+==== Configure Error Tolerance
++
+The default readiness probe is configured to be error-tolerant. If a configuration error occurs (for example, if you misconfigure a policy), the readiness state still changes to `true` once the API instances and policies are deployed. This enables Flex Gateway to scale even though there might be configuration errors.
++
+You can also configure the readiness probe to be error-intolerant, meaning that misconfigured gateway instances never reach a readiness state of `true`.
++
+To configure the Helm chart for error intolerance, remove the following flags from the chart: `allow-api-errors`, `allow-envoy-errors`, and `allow-policy-errors`. In deployments that query the readiness state through HTTP requests, use query parameters: `allowEnvoyErrors=false`, `allowAPIErrors=false`, and `allowPolicyErrors=false`.
++
+[discrete]
+==== Configure External Components
++
+You configure your external components to query the readiness state. For example, in AWS virtual machines that run behind a load balancer, configure the readiness query as a “health check”. To expose the readiness API to external components, apply the following configuration to the Flex Gateway instance:
++
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+ name: probe
+spec:
+ probe:
+   enabled: true
+   port: 3000
+----
```

Comment:
There is one thing missing in the doc I sent you, and that is how to query the readiness probe state through the api once the user exposes the probe with this configuration. They should configure their external components to make a `GET` request to `http://<flex-gateway-url>:<port-configured>/probes/readiness?allowEnvoyErrors=<true|false>&allowAPIErrors=<true|false>&allowPolicyErrors=<true|false>`